src/Main.java

```diff
@@ -1,5 +1,55 @@
+import java.time.LocalDate;
+
 public class Main {
+
+    public static void checksYear(int year) { // объявление метода для проверки является ли год високосным
```

Comment:
```suggestion
    public static void printIsLeapYearOrNot(int year) { // объявление метода для проверки является ли год високосным
```
всё-таки метод выводит, а не только проверяет